db/migrate/20200327142848_create_caution_flag_rules.rb

```diff
@@ -0,0 +1,20 @@
+class CreateCautionFlagRules < ActiveRecord::Migration[5.2]
+  def up
+    create_table :caution_flag_rules do |t|
+      t.integer :rule_id
+      t.string :title
+      t.string :description
+      t.string :link_text
+      t.string :link_url
+      t.timestamps
+    end
+
+    add_foreign_key :caution_flag_rules, :rules, validate: false
+  end
+
+  def down
+    drop_table :caution_flag_rules
+
+    remove_foreign_key :caution_flag_rules, :rules
```

Comment:
If you try to rollback you get an error.  If you're dropping the table, you don't need to also manually remove the foreign key.

Reply:
oh cause the order is wrong, removed manually removing the foreign key